etc/sysconfig/pacemaker.in

```diff
@@ -305,6 +305,16 @@
 # Default: PCMK_cib_timeout=""
 
 
+## Cluster type
+
+# PCMK_cluster_type (Advanced Use Only)
+#
+# Specify the type of cluster layer. Currently only corosync is supported, and
+# pacemakerd overrides this environment variable by setting it to "corosync".
```

Comment:
Specify the cluster layer to be used. If unset, Pacemaker will detect and use a supported cluster layer, if available. Currently, "corosync" is the only supported cluster layer. If multiple layers are supported in the future, this will allow overriding Pacemaker's automatic detection to select a specific one.